readme.txt

```diff
@@ -7,73 +7,79 @@ Requires PHP:      5.4
+* **Analytics:** Explore how users navigate your site and track goals you've set up for your users to complete.
+* **AdSense:** Keep track of how much your site is earning you.
+* **PageSpeed Insights:** See how your pages perform compared to other real-world sites. Improve performance with actionable tips from PageSpeed Insights.
+* **Tag Manager:** Use Site Kit to easily set up Tag Manager- no code editing required. Then, manage your tags in Tag Manager.
+* **Optimize:** Use Site Kit to easily set up Optimize- no code editing required. Then, set up A/B tests in Optimize.
 
 == Installation ==
 
 = Installation from within WordPress =
 
-1. Visit 'Plugins > Add New'.
-2. Search for 'Site Kit by Google'.
+1. Visit **Plugins > Add New**.
+2. Search for **Site Kit by Google**.
 3. Install and activate the Site Kit by Google plugin.
 
 = Manual Installation =
 
 1. Upload the entire `google-site-kit` folder to the `/wp-content/plugins/` directory.
-2. Visit 'Plugins'.
+2. Visit **Plugins**.
 3. Activate the Site Kit by Google plugin.
 
 = After Activation =
 
-1. Visit the new 'Site Kit' menu.
-2. Follow the instructions in the setup flow. If you have multiple Google accounts, make sure to select the one with which Search Console and/or other Google products for your website are already configured.
-3. After completing the setup steps, go to the main dashboard which will already display key metrics from Search Console.
-4. Connect additional Google services under 'Site Kit > Settings' to see more stats.
+1. Visit the new **Site Kit** menu.
+2. Follow the instructions in the setup flow.
+3. Go to the main Site Kit dashboard which already displays key metrics from Search Console.
+4. Connect additional Google services under **Site Kit > Settings**.
 
 == Frequently Asked Questions ==
 
-For the detailed product FAQ and user documentation, please visit the [official Site Kit website](https://sitekit.withgoogle.com/faq/).
+For more information, visit the [official Site Kit website](https://sitekit.withgoogle.com/documentation/).
 
 = Where should I submit my support request? =
 
-For support requests, please use the [wordpress.org support forums](https://wordpress.org/support/plugin/google-site-kit). If you have a technical issue with the plugin where you already have more insight on how to fix it, you can also [open an issue on GitHub instead](https://github.com/google/site-kit-wp/issues).
+First, check our [troubleshooting guide](https://dev-site-kit.pantheonsite.io/documentation/fix-common-issues/). If you're still experiencing issues, use the [wordpress.org support forums](https://wordpress.org/support/plugin/google-site-kit). If you have a technical issue with the plugin where you already have more insight on how to fix it, you can also [open an issue on GitHub instead](https://github.com/google/site-kit-wp/issues).
```

Comment:
replace link to the staging site with the prod link

Reply:
Fixed!